website/src/main/tut/docs/rules/Disable.md

```diff
@@ -27,3 +27,36 @@ Disable.symbols = [
   "scala.Any.asInstanceOf"
 ]
 ```
+
+## Custom error messages
+
+_Since 0.5.4_
```

Comment:
I think adding the version number is a good idea, we should try to use this for all new features.